signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/dbDriver/dbConfig.js

```diff
@@ -0,0 +1,67 @@
+const tracer = require('../tracer');
+
+
+const poolConfig = {host: 'localhost',
+  user: 'admin',
+  database: 'mysql_db',
+  password: 'password',
+  waitForConnections: true,
+  connectionLimit: 10,
+  queueLimit: 0,
+};
+
+
+function initDB(pool) {
+  const initSpan = tracer.startSpan('initDB');
+
+  // Using the Scope's activate() function, we establish
+  // initSpan as the active span for all asynchronous context
+  // stemming from this anonymous function.
+  tracer.scope().activate(initSpan, () => {
+    //    pool = mysql2.createPool(config.poolConfig)
+    //    poolPromise = pool.promise();
+
+    tracer.scope().activate(initSpan, () => {
```

Comment:
Are this second activation and above comments intentional?  You are also finishing the initSpan immediately after this, which probably shouldn't happen until the table is successfully created.

Reply:
Thanks for catching that. Updated it.